source/octf/trace/parser/ParsedIoTraceEventHandler.cpp

```diff
@@ -530,4 +532,174 @@ void ParsedIoTraceEventHandler::getFilePath(uint64_t devId,
     }
 }
 
+class ParsedIoTraceEventHandler::FileSystemViewer : public IFileSystemViewer {
+public:
+    FileSystemViewer(uint64_t partId, const std::set<FileName> &fsNames)
+            : IFileSystemViewer()
+            , m_partId(partId)
+            , m_fsNames(fsNames) {}
+
+    virtual std::string getBaseName(uint64_t id) const override {
+        std::string basename = "";
+
+        FileName key;
+        key.Id = id;
+        key.DeviceId = m_partId;
+
+        auto iter = m_fsNames.find(key);
+        if (iter != m_fsNames.end()) {
+            auto i = iter->Name.rfind('.', iter->Name.length());
```

Comment:
I think the second argument is redundant (it defaults to npos then - which should have the same behaviour)